static/app/components/events/interfaces/frame/deprecatedLine.tsx

```diff
@@ -270,9 +278,35 @@ export class DeprecatedLine extends Component<Props, State> {
     return null;
   }
 
+  renderShowHideToggle(onShowFramesToggle) {
+    const hiddenFrameCount = this.props.hiddenFrameCount;
+    if (hiddenFrameCount) {
+      return (
+        <a
```

Comment:
`a` tags should only be used for links. This should semantically be a button since it doesn't navigate. We should also use our reusable `<Button />` component for this.